src/connector/src/parser/common.rs

```diff
@@ -95,12 +95,7 @@ fn do_parse_simd_json_value(
         // debezium converts time to i64 for mysql and postgres
         DataType::Time => match v {
             BorrowedValue::String(s) => str_to_time(s).map_err(|e| anyhow!(e))?.into(),
-            BorrowedValue::Static(_) => Time::with_milli(
-                ensure_i64!(v, i64)
-                    .try_into()
-                    .map_err(|_| anyhow!("cannot cast i64 to time, value out of range"))?,
-            )?
-            .into(),
+            BorrowedValue::Static(_) => Time::with_micro(ensure_i64!(v, i64))?.into(),
```

Comment:
Is this fix only apply to debeizum json or all format? You may need to check the `format` first.